index.html

```diff
@@ -4,24 +4,37 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>My Portfolio Mobile Version</title>
+    <title>My Portfolio Website</title>
+    <!-- Link to the Font-Awesome Font -->
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.2/css/font-awesome.min.css"
+    />
     <link rel="stylesheet" href="style.css" />
   </head>
 
   <body>
-    <header id="headline">
-      <nav class="nav-container">
-        <div class="logo-text"><a href="#">AlierPM</a></div>
-        <div class="right-logo">
-          <img src="images/menu-icon.png" alt="image" />
-        </div>
-      </nav>
+    <header class="mobo">
+      <h2 class="logo-text"><a href="#">AlierPM</a></h2>
+      <div class="nav-menu">
+        <ul class="nav-flex">
+          <li class="nav-link">
+            <a href="#">Portfolio</a>
+          </li>
+          <li class="nav-link">
+            <a href="#">About</a>
+          </li>
+          <li class="nav-link">
+            <a href="#">Contact</a>
+          </li>
+        </ul>
+        <i class="mobo-menu fa fa-bars"></i>
+      </div>
     </header>
     <main>
-      <div class="headline-section grid">
-        <div class="bg-img"></div>
+      <section class="headline-section">
         <div class="middle-text">
-          <h1>I'm Alier Glad to see you.</h1>
+          <h1>I'm Alier <br />Glad to see you!</h1>
```

Comment:
- [ ] Great job making the header section but I noticed here that the font weight for the `h1` tag is not the same with the Figma template, so please Fix it to be the same with the Figma 🙏🏻🙏🏻